### ISSUE TITLE
TMDB API issues

### ISSUE THREAD

**Issue by 4getit** [Radarr/Radarr#1145]
TMDB is throwing random 502 bad gateway errors today. The error message is HTML so it causes JSON parsers to fail.
Maybe it's an idea to do a quick check on the response header and then retry? Right now, it's failing halfway list imports with:

```
Request Failed: Unexpected character encountered while parsing value: <. Path '', line 0, position 0.

Newtonsoft.Json.JsonReaderException: Unexpected character encountered while parsing value: <. Path '', line 0, position 0.
  at Newtonsoft.Json.JsonTextReader.ParseValue () <0x41b71930 + 0x0057b> in <filename unknown>:0 
  at Newtonsoft.Json.JsonTextReader.ReadInternal () <0x41b715f0 + 0x00043> in <filename unknown>:0 
  at Newtonsoft.Json.JsonTextReader.Read () <0x41b715a0 + 0x0001b> in <filename unknown>:0 
  at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.ReadForType (Newtonsoft.Json.JsonReader reader, Newtonsoft.Json.Serialization.JsonContract contract, Boolean hasConverter) <0x41b713b0 + 0x00084> in <filename unknown>:0 
  at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.Deserialize (Newtonsoft.Json.JsonReader reader, System.Type objectType, Boolean checkAdditionalContent) <0x41b675c0 + 0x000e3> in <filename unknown>:0 
```

**Comment by spencercharest** [comment]
I'm encountering this issue with IMDB lists as well.

**Comment by galli-leo** [comment]
I fixed the list import stopping due to such errors. Won't retry the movie yet, but that should happen when the next list import sync occurs. Therefore, I don't think it's necessary to retry the movie instantly.

**Comment by 4getit** [comment]
Great! It's working and it fixed #1193 and #1294 as well.
